### DISCUSSION TITLE
Client to play on the test server

### DISCUSSION THREAD

**Discussion by Joahim49** [fedorovvl/tso_client#145] [Q&A] [answered]
Hello, Is it possible to play on the test server?
If you can play, please answer.

Maybe the "client.swf" file is enough, if so, where to download it or how to create it yourself?

I greet the authors and thank you for their excellent work. ;))

**Comment by fedorovvl** [comment, marked as answer]
check latest release

**Reply by Joahim49** [reply]
Thank you vvvveeeeerrrrrry much :) Good job ;) <beer 4 u>
